src/richie/apps/search/indexers/categories.py

```diff
@@ -26,7 +27,7 @@ class CategoriesIndexer:
     """
 
     document_type = "category"
-    index_name = "richie_categories"
+    index_name = f"{ES_INDICES_PREFIX}categories"
```

Comment:
May be nicer to write `f"{ES_INDICES_PREFIX}_categories"` with ES_INDICES_PREFIX = "richie".